app/project/[id]/components/project-files.tsx

```diff
@@ -107,6 +140,23 @@ const filesColumns: Tan.ColumnDef<DesignFile>[] = [
       );
     },
     size: 30,
+    enableSorting: false,
+  },
+  {
+    accessorKey: "id",
+    header: () => <div className="flex justify-center">Delete file</div>,
```

Comment:
sometimes this seems to wrap, maybe these two columns don't need a header as the icon is self explanatory
![image](https://github.com/patrickchin/haru2-vercel/assets/1392122/e622a948-7893-436e-a9dc-1b9bee098bf8)

then maybe we could have a tool tip on the icons to show popup, the shadcn tooltip seem so excessive ... maybe we don't need it 
https://ui.shadcn.com/docs/components/tooltip